x-pack/plugins/actions/server/builtin_action_types/jira/translations.ts

```diff
@@ -9,3 +9,19 @@ import { i18n } from '@kbn/i18n';
 export const NAME = i18n.translate('xpack.actions.builtin.case.jiraTitle', {
   defaultMessage: 'Jira',
 });
+
+export const WHITE_LISTED_ERROR = (message: string) =>
```

Comment:
we are working on remove `whitelist` phrase from our plugins, so, please rename it to `ALLOWED_HOSTS_ERROR` and `apiAllowedHostsError`